src/docs/Dedication.adoc

```diff
@@ -2,11 +2,10 @@
 [discrete]
 == Legal notice
 
-The Activeeon team (C) 2016 by Activeeon
+The Activeeon SAS, © 2007-2019. All Rights Reserved.
```

Comment:
Could you make it auto-calculating the current year? Otherwise it will be always forgotten to update.

Reply:
Good point, I'll see what I can do

Reply:
Suggestion applied